tools/generator/go_mod_tidy_hack.go

```diff
@@ -1,3 +1,4 @@
+//go:build modhack
 // +build modhack
```

Comment:
@jhendrixMSFT could you please take a look at this change? Is this expected as in go 1.17?
And should we do the same for all other released SDK modules in their next release?